src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -23,7 +23,15 @@ const fileObj = Yup.object().shape<SchemaShape<ImgLink>>({
 //construct strict shape to avoid hardcoding shape keys
 const shape: SchemaShape<FormValues> = {
   categories_sdgs: Yup.array().min(1, "required"),
-  tagline: requiredString,
+  tagline: Yup.string().test(
```

Comment:
```suggestion
  tagline: requiredString.max(140, "max length is 140 chars")
```
no need for `.test` since `string` has `max` method

could also show curr length 
```ts
requiredString.max(140, ({ max }) => `${max} / 140`)
```

this would show red text 
----------------------
[__________________]
                         156/140